plugins/chat/assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -25,8 +25,10 @@ import {
   chatComposerButtons,
   chatComposerButtonsDependentKeys,
 } from "discourse/plugins/chat/discourse/lib/chat-composer-buttons";
+import { mentionRegex } from "pretty-text/mentions";
 
 const THROTTLE_MS = 150;
+const MENTION_DEBOUNCE_MS = 1000;
```

Comment:
Yes given this delay, we absolutely want to use `discourseDebounce`, otherwise some tests could be delayed by 1000m